website/www/site/layouts/partials/section-menu/en/get-started.html

```diff
@@ -22,12 +22,13 @@
     <li><a href="/get-started/quickstart-go/">Quickstart - Go</a></li>
   </ul>
 </li>
+<li><a href="/get-started/from-spark/">From Apache Spark</a></li>
```

Comment:
The actual Overview page has a Getting Started section you can include this new doc in.

Reply:
Thanks, I just added a link there too